styles.css

```diff
@@ -1,785 +1,862 @@
+        font-family: 'Poppins', sans-serif;
+        font-style: normal;
+        font-weight: 500;
+        font-size: 1.95vw;
+        line-height: 5vw;
+        display: flex;
+        align-items: center;
+        text-align: center;
+        color: #344563;
+        flex: none;
+        order: 0;
+        flex-grow: 0;
+        margin-left: 5vw;
+    }
+    .page{
+        display: grid;
+        grid-template: 100% 100%;
+        background: #E5E5E5;
+        gap: 10vw;
+    }
+    .mainpage{
+        display: flex;
+        flex-direction: column;
+        width: 100%;
+        height: 100%;
+        background: white;
+        border-bottom-left-radius: 64px;
+        background-image: url("topback.png");
+        background-size: 99vw 25vw;
+        background-repeat: no-repeat;
+        background-position: 0.5vw 2vw;
+    }
+    .mainheader{
+        width: 30%;
+        height: 3.25vw;
+        font-family: 'Poppins', sans-serif;
+        font-style: normal;
+        font-weight: 700;
+        font-size: 3.5vw;
+        line-height: 4.25vw;
+        display: flex;
+        align-items: center;
+        letter-spacing: 0.37px;
+        color: #091E42;
+        flex: none;
+        order: 0;
+        flex-grow: 0;
+        padding-left: 15vw;
+        padding-top: 20vw;
+    }
+    .mainpara{
+        width: 68%;
+        height: 7.5vw;
+        font-family: 'Poppins', sans-serif;
+        font-style: normal;
+        font-weight: 400;
+        font-size: 1.65vw;
+        line-height: 3.5vw;
+        color: #344563;
+        flex: none;
+        order: 1;
+        flex-grow: 0;
+        padding-left: 15vw;
+    }
+    .let{
+        width: 50%;
+        height: 1vw;
+        font-family: 'Poppins', sans-serif;
+        font-style: normal;
+        font-weight: 500;
+        font-size: 2.08vw;
+        line-height: 3.5vw;
+        color: #7F8CFF;
+        flex: none;
+        order: 1;
+        flex-grow: 0;
+        padding-left: 10vw;
+    }
+    .mainpagelink{
+        width: 40vw;
+        display: flex;
+        flex-direction: row;
+        flex-wrap: nowrap;
+        order: 2;
+        flex-grow: 0;
+    }
+    .bottom{
+      width: 68vw;
+      height: 20vw;
+      margin-left: 10vw;
+      background-image: url("bottoback.png");
+      background-size: 68vw 15v;
+      background-repeat: no-repeat;
+      order: 2;
+      flex-grow: 0;
+    }
+    .can{
+        display: flex;
+        flex-grow: 0;
+        width: 50%;
+        height: 1vw;
+        margin-left: 3.5vw;
+        }
+    .card{
+        box-sizing: border-box;
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        padding: 16px;
+        gap: 12px;
+        width: 95vw;
+        height: 100%;
+        background: #FFFFFF;
+        border: 1px solid #DFE1E6;
+        border-radius: 16px;
+        flex: none;
+        order: 0;
+        flex-grow: 0;
+        margin-left: 1vw;
+        margin-right: 1vw;
+    }

```

Comment:
- [ ] It seems like the width of your project card is a bit too wide and it's not in sync with the Figma design so please make sure to reduce the width so it looks more similar to the Figma design
- Your design:
  ![firefox_uWySvKqG3K](https://github.com/feyisakedir/Portfolio-2nd-Project/assets/104968236/3be131bb-efc8-4e2d-8b93-3111c7add981)

- Figma design:
  ![firefox_gpUvfLrCte](https://github.com/feyisakedir/Portfolio-2nd-Project/assets/104968236/0773cad7-5bc9-4de0-ab94-cedd991a18e5)
- [ ] Some of the text in the project cards are also overflowing and causing a bad UI so please make sure to fix them.
- Here's how it looks:
  ![image](https://github.com/feyisakedir/Portfolio-2nd-Project/assets/104968236/77aa50c5-86de-4dc5-8472-bad35d2a2575)